etc/tikv.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=TiKV, a distributed transactional key value store.
```

Comment:
seems this Description discription should be something like `TiKV Service daemon`

Reply:
Agreed!